rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1603,17 +1612,24 @@ def test_remove_blockchain_accounts(
         btc_balances=['3000000', '5000000'],
     )
     # remove the new BTC account
-    response = requests.delete(api_url_for(
-        rotkehlchen_api_server,
-        'blockchainsaccountsresource',
-        blockchain='BTC',
-    ), json={'accounts': [UNIT_BTC_ADDRESS1], 'async_query': async_query})
-    if async_query:
-        task_id = assert_ok_async_response(response)
-        wait_for_async_task(rotkehlchen_api_server, task_id)
-    else:
-        assert_proper_response(response)
-    assert rotki.chain_manager.accounts.btc == btc_accounts_after_removal
+    with ExitStack() as stack:
+        setup.enter_blockchain_patches(stack)
+        response = requests.delete(api_url_for(
+            rotkehlchen_api_server,
+            "blockchainsaccountsresource",
+            blockchain='BTC',
```

Comment:
With lefteris you commented that was better to use the hardcoded value or the enum.value?